elixir/apps/web/test/web/live/settings/identity_providers/openid_connect/new_test.exs

```diff
@@ -143,10 +143,22 @@ defmodule Web.Live.Settings.IdentityProviders.OpenIDConnect.NewTest do
         }
       )
 
-    validate_change(form, %{provider: %{name: String.duplicate("a", 256)}}, fn form, _html ->
+    changed_values = %{
+      provider: %{
+        name: String.duplicate("a", 256),
+        adapter_config: %{provider_attrs.adapter_config | "client_id" => ""}
+      }
+    }
+
+    validate_change(form, changed_values, fn form, html ->
       assert form_validation_errors(form) == %{
-               "provider[name]" => ["should be at most 255 character(s)"]
+               "provider[name]" => ["should be at most 255 character(s)"],
+               "provider[adapter_config][client_id]" => ["can't be blank"]
              }
+
+      assert html
```

Comment:
Isn't this assertion duplicates the one above?

Reply:
I was thinking the one above was on the Elixir struct rather than the HTML.  When I was troubleshooting the issue I noticed that the errors were in the struct, but didn't show up on the page, so this was just to make sure that the error showed up on the page.  Happy to remove it if it's not necessary though.

Reply:
It's taken from HTML pretty much the same as you do here (check `form_validation_errors/1` implementation).

Reply:
🤦‍♂️ Sorry about that.  I probably should have checked that function first before making the assumption.  I'll remove the unnecessary checks.

Reply:
Updated